OCAPP/Controllers/Conferences.py

```diff
@@ -76,7 +76,7 @@ def register_user(conference_id):
 			for message in field:
 				flash(message, field)
 		return redirect('/')
-
+	print data 
```

Comment:
Let's remove print statments before I merge it.  For any comments I make, address them, commit and push and these will automatically be updated.